### PR TITLE
git*: add Turkish translation

### PR DESCRIPTION
**Pull request by yutyo**
- [ ] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages.tr/common/git-apply.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# git apply
+
+> Apply a patch to files and/or to the index.
```
Comment (navarroaxel) [inline pages.tr/common/git-apply.md:3]: This is English.

File: pages.tr/common/git-check-ref-format.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# git check-ref-format
+
+> Girilen refname kabul edilebilir mi yoksa değil mi diye kontrol eder, ve eğer kabul edilemezse sıfır olmayan bir çıktı verir.
+> Daha fazla bilgi için: <https://git-scm.com/docs/git-check-ref-format>.
+
+- Belirtilen refname'in biçimini kontrol et:
+
+`git check-ref-format {{refs/head/refname}}`
```
Comment (navarroaxel) [inline pages.tr/common/git-check-ref-format.md:8]: `refname` could be translated.

Comment (yutyo) [reply pages.tr/common/git-check-ref-format.md]: done

File: pages.tr/common/git-revert.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# git revert
```
Comment (sbrl) [inline pages.tr/common/git-revert.md:1]: This page appears to be in english?

File: pages.tr/common/git-show-branch.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+# git show-branch
```
Comment (sbrl) [inline pages.tr/common/git-show-branch.md:1]: And this one

Comment (yutyo) [reply pages.tr/common/git-show-branch.md:1]: done

File: pages.tr/common/git-revert.md (hunk 3 above)
Comment (yutyo) [reply pages.tr/common/git-revert.md:1]: done

**Comment by yutyo** [comment]
Hope its done now.

**Review by CleanMachine1 — APPROVED** [review]

File: pages.tr/common/git-send-email.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+`git send-email -1 {{commit}}`
+
+- Mevcut dizindeki belirtilen sayı kadar (örneğin 10) commit'i gönder:
+
+`git send-email {{-10}}`
+
+- Gönderilecek yama serisi için bir giriş e-posta mesajı gönder:
+
+`git send-email -{{commit}} --compose`
```
Comment (navarroaxel) [inline pages.tr/common/git-send-email.md:21]: The `commit` placeholder should be `number_of_commits`. Perhaps
```suggestion
`git send-email -{{commits_sayı}} --compose`
```

File: pages.tr/common/git-send-email.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+- Gönderilecek her bir yama için e-posta mesajını görüntüle ve düzenle:
+
+`git send-email -{{commit}} --annotate`
```
Comment (navarroaxel) [inline pages.tr/common/git-send-email.md:25]: The `commit` placeholder should be `number_of_commits`. Perhaps
```suggestion
`git send-email -{{commits_sayı}} --annotate`
```

File: pages.tr/common/git-rev-list.md [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+
+- Mevcut daldaki tüm commit'leri sırala:
+
+`git rev-list {{HEAD}}`
+
+- Belirtilen daldaki belirtilen tarihten daha yakın olan commit'leri sırala:
+
+`git rev-list --since={{'2019-12-01 00:00:00'}} {{dal_ismi}}`
+
+- Belirtilen commit'deki tüm birleştirme commit'lerini sırala:
```
Comment (navarroaxel) [inline pages.tr/common/git-rev-list.md:14]: This English page has 4 examples instead of 4.

**Review by sbrl — APPROVED** [review]
Looks ok to me after @navarroaxel's comments are addressed. Thanks, @yutyo!

**Comment by navarroaxel** [comment]
@yutyo you ignored my last comments 😑

File: pages.tr/common/git-send-email.md (hunk 5 above)
Comment (yutyo) [reply pages.tr/common/git-send-email.md:21]: Its OK as this as well.

**Pull request merged**